core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -267,4 +267,11 @@ public interface Configs {
           .version(ConfigConstants.VERSION_0_5_0)
           .longConf()
           .createWithDefault(60 * 60 * 1000L);
+
+  ConfigEntry<Long> METRICS_TIME_SLIDING_WINDOW_SECONDS =
+      new ConfigBuilder("gravitino.metrics.timeSlidingWindowSecs")
```

Comment:
Do we need `Secs` or `Sec`?

Reply:
I prefer to using `Secs` to keep consistent with current `gravitino.lock.cleanIntervalInSecs`, WDYT?

Reply:
OK.